packages/lti-client/src/libs/post_message/resize_iframe.ts

```diff
@@ -0,0 +1,21 @@
+import { PostMessageClient } from "./client";
+
+export class ResizeIframe {
+  #client: PostMessageClient;
+
+  constructor(client?: PostMessageClient) {
+    this.#client = client ?? new PostMessageClient();
+  }
+
+  async isSupported(): Promise<boolean> {
+    const capabilities = await this.#client.getCapabilities();
+    return capabilities.some((cap) => cap.subject === "lti.frameResize");
```

Comment:
This was traditionally Canvas specific but there's a chance other LMSs support it now.

Reply:
It's mostly Canvas specific, but I know for a fact that Brigthspace does support it